pyproject.toml

```
@@ -23,7 +23,7 @@ dependencies = [
     "solara>=1.40.0",
     "pyyaml>=5.4.1",
     "specutils>=1.18",
-    "specreduce>=1.4.1",
+    "specreduce @ git+https://github.com/astropy/specreduce.git@refs/pull/253/head#egg=specreduce",
```

this should be unblocked now if you want to pin 1.5

```suggestion
    "specreduce>=1.5",
```